packages/twenty-server/src/core/quick-actions/intelligence.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@nestjs/common';
+
+import axios from 'axios';
+
+import { EnvironmentService } from 'src/integrations/environment/environment.service';
+
+@Injectable()
+export class IntelligenceService {
+  constructor(private readonly environmentService: EnvironmentService) {}
+
+  async enrichCompany(domainName: string): Promise<any> {
```

Comment:
@FelixMalfait Should be better if we can `type` the return of this endpoints

Reply:
Any idea where I can get this type from? Do we have to created a dedicated one at the top of the file? Or where would you put it?
That would be a JSON representing the company object, like {name: ..., domainName: ...}

Reply:
I the object doesn't exist I rather create an `interfaces` folder inside this module with the dedicated types/interfaces, It's better to have one file per type/interface